### ISSUE TITLE
ios: delete not working since we moved to 1.1.0

### ISSUE THREAD

**Issue by ldikmans** [capacitor-community/http#153]
**Describe the bug**
Delete gives an error since we upgraded the plugin to version 1.1.0: 
we get the following error message: ERROR MESSAGE:  {"errorMessage":"De bewerking kan niet worden voltooid. (NSURLErrorDomain fout -1020.)","message":"De bewerking kan niet worden voltooid. (NSURLErrorDomain fout -1020.)"}

translated: 
ERROR MESSAGE:  {"errorMessage":"Operation could not be completed. (NSURLErrorDomain error -1020.).","message":"Operation could not be completed. (NSURLErrorDomain error -1020.)"}

**To Reproduce**
1. Create a call with the plugin 1.0.0  that uses delete with no body
2. Run it -> success
3. Upgrade to 1.0.0 

All other calls are still working (post, get, put)

**Expected behavior**
successful call

 
         ```
import { HttpHeaders, Http } from '@capacitor-community/http';

 const httpHeaders = { 'content-type': 'application/json' };
 const   result = await Http.del({
                url: url,
                headers: httpHeaders
            });
        }
```

**Comment by joeflateau** [comment]
fixed by my PR #152 waiting to be merged

**Comment by thomasvidas** [comment]
Fixed in #152
